backend/constants.py

```diff
@@ -211,5 +211,4 @@
 
 SERVICE_PROVIDER_FIELDS = ['name', 'website', 'report_year', 'report_link', 'expenses', 'client_total', 'staff_total', 'is_bookmarked', 'notes']
 
-FIREBASE_AUTH_FILE_PATH = os.getenv("PROJECTROOT") + "/instance/c4k-dashboard-firebase-adminsdk-ypbc3-c66b8c5a1c.json"
```

Comment:
Does this need to be changed? Environment var `PROJECTROOT` depends on where you run your `pipenv` commands. I usually run it from the root dir but it seems you run it from the frontend directory? I prob shouldn't have used env vars for this in hindsight lol.